### PR TITLE
buku: 2.8 -> 2.9

### PR DESCRIPTION
**Pull request by infinisil**
###### Motivation for this change

Updated the package while fixing the executable not running due to missing python modules.

###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/buku/default.nix [hunk 1]
```diff
@@ -2,21 +2,22 @@
...
   };
 
-  buildInputs = [
+  propagatedBuildInputs = [
     cryptography
     beautifulsoup4
+    requests2
+    urllib3
   ];
-  propagatedBuildInputs = [ beautifulsoup4 ];
```
Comment (Mic92) [inline pkgs/applications/misc/buku/default.nix]: According to https://github.com/jarun/Buku/blob/master/setup.py#L34 these are all run-time dependencies. Does this also needs to be fixed in 17.03?

Comment (Infinisil) [inline pkgs/applications/misc/buku/default.nix]: Before this commit it didn't work in 17.03, so yes it does. It's my first PR here, sorry if I missed something :)

File: pkgs/applications/misc/buku/default.nix [hunk 2]
```diff
@@ -31,7 +32,7 @@ with pythonPackages; buildPythonApplication rec {
     homepage = https://github.com/jarun/Buku;
     license = licenses.gpl3;
     platforms = platforms.linux;
-    maintainers = with maintainers; [ matthiasbeyer ];
+    maintainers = with maintainers; [ matthiasbeyer infinisil ];
```
Comment (matthiasbeyer) [inline pkgs/applications/misc/buku/default.nix]: Hey! You don't need to put yourself into the maintainers array, if you don't want to! If you want to, yes, feel free, I appreciate that! But I want you to know that you do not have to! :smile:

Comment (Infinisil) [inline pkgs/applications/misc/buku/default.nix]: Well I'd like this package to stay up-to-date, so I guess that's okay 😉. I just got up and running with my NixOS installation and I also intend to contribute much more in the future :D

Comment (matthiasbeyer) [inline pkgs/applications/misc/buku/default.nix]: Awesome! Then welcome to the club!

**Pull request merged**